Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js

```diff
@@ -120,7 +120,11 @@ switch (command) {
         sendRequest('GET','user');
         return 'ok';
     case 'demisto-api-post':
-        var body = JSON.parse(args.body);
+        if(args.body)
+            var body = JSON.parse(args.body);
+        else
+            logDebug('The body is empty.')
+
         return sendRequest('POST',args.uri, args.body);
```

Comment:
@yaakovi 
we parse the body but didn't send the body variable to the request 
is this by mistake?

Reply:
don't think so, please test this behavior

Reply:
checked 
it fail if we send the body var